<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmServerManager</class>
 <widget class="QDialog" name="frmServerManager">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>387</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>387</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WPS 서버 관리</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>356</y>
     <width>100</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>50</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>적용</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="grpList">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>168</y>
     <width>581</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>서버 목록</string>
   </property>
   <widget class="QTableWidget" name="tblServer">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>20</y>
      <width>569</width>
      <height>153</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>488</x>
     <y>356</y>
     <width>100</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>50</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="grpWPS">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>8</y>
     <width>581</width>
     <height>153</height>
    </rect>
   </property>
   <property name="title">
    <string>WPS 서버 정보</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>20</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>이름</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtName">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>16</y>
      <width>525</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>48</y>
      <width>37</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>URL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>76</y>
      <width>37</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>버전</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtURL">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>44</y>
      <width>525</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtVersion">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>72</y>
      <width>525</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1.0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>104</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>서버 유형</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="optGeneral">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>104</y>
      <width>109</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>General WPS</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="optKopss">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>104</y>
      <width>145</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>KOPSS API Support</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>372</x>
      <y>124</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>추가</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelete">
    <property name="geometry">
     <rect>
      <x>476</x>
      <y>124</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>삭제</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tblServer</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
